Example/PaymentSheet Example/PaymentSheet Example/CustomerSheetTestPlayground.swift

```diff
@@ -184,6 +185,8 @@ struct CustomerSheetPaymentOptionView: View {
                 .foregroundColor(.black)
             Text(paymentOptionDisplayData?.displayData().label ?? "None")
                 .accessibility(identifier: "Payment method")
+                .foregroundColor(colorScheme == .dark ? .white : .black)
+
         }.padding()
             .foregroundColor(.black)
```

Comment:
Could you set this line to `.foregroundColor(.primary)` instead to reflect the system default? Checking the color scheme manually shouldn't be necessary